test-partner/debugpartner.yaml

```diff
@@ -51,6 +48,7 @@ spec:
       securityContext: {}
       serviceAccount: default
       serviceAccountName: default
+      automountServiceAccountToken: true
```

Comment:
we don't need access to k8s API, so I guess this can be set to false.

Reply:
Fixed in the next commit, thanks!